Demo/AuthenticatorDemo/ViewController+WordPressAuthenticationDelegate.swift

```diff
@@ -90,3 +88,46 @@ extension ViewController: WordPressAuthenticatorDelegate {
         print(error)
     }
 }
+
+extension ViewController {
+
+    // This is just so we can avoid nesting within a dismiss block and the weak self dance.
+    //
+    // See WordPress iOS
+    //
+    // - WordPressAuthenticationManager sync(credentials:, onCompletion:)
+    // - WordPressAuthenticationManager syncWPCom(authToken:, isJetpackLogin:, onCompletion:)
+    // - AccountService createOrUpdateAccountWithAuthToken:success:failure:
+    private func sync(credentials: AuthenticatorCredentials, onSynced: @escaping () -> Void) {
```

Comment:
This `onSynced` is not called anywhere.

Reply:
Good eyes!

Addressed in d1b6c91f

Thanks 🙇‍♂️